XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -313,6 +310,21 @@ public override void Destroy()
 
         #endregion IMixedRealitySerivce Implementation
 
+        private void EnsureCameraRigSetup()
```

Comment:
Should we add a menu item to setup rig manually like you are requesting in #911

Reply:
We "could" but that would be counter productive at this time, the automatic validation has been retained.

And the MixedRealityToolkit -> Configure option in the menu performs the manual setup already.